main.js

```diff
@@ -92,24 +92,36 @@ function getMainMapAddrs(json) {
     return addrs;
 }
 
-function drawMainMap(addrList) {
-    let map = L.map('main-map', {
-        center: [44.4781994, -73.2126357],
-        zoom: 20
-    });
+async function drawMainMap(addrList) {
+    let map = L.map('main-map').setView([44.4781994, -73.2126357],15);
 
     L.tileLayer('https://{s}.tile.openstreetmap.fr/osmfr/{z}/{x}/{y}.png', {
         maxZoom: 20,
         attribution: '&copy; Openstreetmap France | &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
     }).addTo(map);
 
-    addrList.forEach((ad) => {
-        const latArr = getCoords(ad.addr);
 
-        console.log(latArr);
-        let mark = L.marker([latArr.lat, latArr.lng]).addTo(map);
-        return mark;
-    })
+    for await (const ad of addrList) {
+        const latArr = await getCoords(ad.addr);
+
+        console.log({latArr});
+        let { lat, lon } = latArr;
```

Comment:
here, I destructure the ```latArr``` variable so that I have two variables called ```lat``` and ```lon``` which are properties that ```latArr``` will have after ```getCoords``` returns a value.